sdk/daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Upgrading.scala

```diff
@@ -334,9 +344,10 @@ object TypecheckUpgrades {
   private def checkDeleted[K, V](
       arg: Upgrading[Map[K, V]],
       handler: (K, V) => UpgradeError.Error,
+      ignore: (K, V) => Boolean = (_: K, _: V) => true,
```

Comment:
I find "ignore" misleading because to me `ignore(x) == true` sounds like x should be ignored.

Reply:
I've replaced it with `filter`